--- 03_disk_usage_original.py	2025-08-07 15:29:16.709874248 +0530
+++ 03_disk_usage_fixed.py	2025-08-07 15:31:05.404965123 +0530
@@ -10,7 +10,7 @@
     gigabytes_free =du.free/2**30
     if percent_free < min_percent or gigabytes_free < min_absolute:
         return False
-    return True
+    sys.exit(1)
 
 # Check for at least 2 GB and 10% free
 #if not check_disk_usage("/",2*2**30,10):  # something wrong since we are doing the gigabyte conversion twice --ref Line 10
